src/parser/transforms.js

```diff
@@ -421,14 +426,21 @@ module.exports.convertAst = function convertAst(result, preprocessedResult, visi
       // always reference first part of tag name, this also has the advantage
       // that errors regarding this tag will only mark the tag name instead of
       // the whole tag + children
-      const n = node.parts[0];
+      //
+      // However, with plain <footer> type nodes, there will be no "parts"
```

Comment:
this doesn't look right.
there should **always**. be parts, also for plain nodes. this is for eslint to reference the tag name instead of the whole tag. if an error references the tag, it will mark the whole tag from start to end as an error. e.g. `<footer> many things </footer>` will be marked as an error, in editors it will underline everything red

Reply:
hm -- maybe we should add some tests to keep me honest :sweat_smile:

Reply:
what's a scenario where we would have a big squiggle between the tags?